nanoFirmwareFlasher.Library/EspTool.cs

```diff
@@ -29,7 +29,7 @@ public partial class EspTool
         private readonly string _serialPort = null;
 
         /// <summary>
-        /// The baud rate for the serial port. The default comming from <see cref="Options.BaudRate"/>.
+        /// The baud rate for the serial port. The default comming from <see cref="System.IO.Ports.SerialPort.BaudRate"/>.
```

Comment:
this is coming from the options class, not serial port

Reply:
I did think that... but... given this is a lib, it is not "directly accessible" for docs... Can you think of anything that would work in that regard???

Reply:
You've lost me there... in what regard? 

And yes of course you are correct: this is a tool, not made available on the docs. That's the reason why the docs haven't been completed (and some are outdated). 😉

Reply:
The solution contains 2 projects, one is the tool and one is a library. A warning is generated that Options.BaudRate within the library (because the library does not directly have access to the command line parser). For the moment, I will revert this as can always be corrected later, and as there are considerabily less warnings to contend with now, it is easily spotted.